compiler/src/iree/compiler/Dialect/HAL/Transforms/test/convert_to_hal.mlir

```diff
@@ -53,11 +53,12 @@ util.func public @simpleDispatch(%arg0: !hal.buffer_view, %arg1: !hal.buffer_vie
   %c16 = arith.constant 16 : index
   %c0 = arith.constant 0 : index
 
-  // CHECK: %[[NULL_FENCE:.+]] = util.null : !hal.fence
-
   // CHECK: %[[BUFFER_USAGE:.+]] = hal.buffer_usage<"{{.+}}Transfer{{.+}}Dispatch{{.+}}"> : i32
   // CHECK: %[[MEMORY_TYPE:.+]] = hal.memory_type<"DeviceVisible|DeviceLocal"> : i32
 
+  // CHECK: %[[NULL_FENCE:.+]] = util.null : !hal.fence
+
+
```

Comment:
you can change the 3 checks to `CHECK-DAG` as order is not important, only that they exist